src/components/LibraryItem.tsx

```diff
@@ -184,9 +185,6 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     }
 
     onLibraryItemClicked() {
-
-        // Toggle expansion state.
-        let currentlyExpanded = this.state.expanded;
-        this.setState({ expanded: !currentlyExpanded });
+        LibraryEntryPoint.dispatchEvent("libraryItemClick", this);
```

Comment:
@Benglin Hi Ben, thank you very much for the detailed review.
May I know where should the libraryView.on function be called to register the callback?